azure.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: chat-with-your-data-solution-accelerator
+
+services:
+  Website:
```

Comment:
When you change this it will also need to be changes every where else in code, bicep tags, etc.